src/ConfigInterface.php

```diff
@@ -109,6 +114,11 @@ public function setIndent(string $indent): self;
 
     public function setLineEnding(string $lineEnding): self;
 
+    /**
+     * Set number of processes to use.
+     */
+    public function setParallel(int $parallel): self;
```

Comment:
i am not convinced it should be part of config.
like, the config should say what should be the "constant" for the project - eg which files to fix or using what rules.

how much jobs in parallel we can execute depends on machine of each dev working on the project, CI capabilities etc